components/vf-show-more/vf-show-more.njk

```diff
@@ -0,0 +1,48 @@
+{# Make sure any variables are listed inside the following if statement #}
+{% if context %}
+  {# {% set vf-show-more = context.vf-show-more %} #}
+{% endif %}
+
+{# You can do Nunjucks logic in your templates that references the .yml settings
+   What's Nunjucks: https://mozilla.github.io/nunjucks/templating.html #}
+{#
+{% if item.image == "blank" %}
+  <span><!-- no image --></span>
+  {% else %}
+  {{ item.image }}
+{% endif %}
+#}
+
+<section data-vf-js-show-more data-vf-js-show-more-pager-size="3" class="vf-show-more">
+  {% render '@vf-section-header--default', {
+    "section_title": "Show more demo"
+  } %}
+
+  <div class="vf-grid vf-grid__col-3">
+    {% set items = [
```

Comment:
we could probably set this in the yml as I think `vf-card` is 'ready'

Reply:
As it's just demo data, is it worth extracting?

Reply:
It all depends with how well it plays in 11ty - I've had an issue with one components with this or similar. So I'll check.

Another thing for the checklist of new components -- does it work in 11ty 'as expected'

Reply:
Ah, have actually been developing in the vf-component-library, and all is well there :)

![image](https://user-images.githubusercontent.com/928100/91289939-48741f80-e793-11ea-81c8-4ec354749903.png)

Reply:
Although both 11ty and fractal use nunjucks -- 11ty is a little strict on the whitespace you give it. 

So there is a need for testing the component 'in the real world' because of that. Annoying that it is.